### ISSUE TITLE
Photos do not reflect original EXIF orientation

### ISSUE THREAD

**Issue by 1000miles** [sozialhelden/wheelmap#636]
This is a follow-up of the comment mentioned in ticket https://github.com/sozialhelden/wheelmap/issues/167#issuecomment-311608771.

At the moment when we upload images manually through the browser the images are shown as expected. But when we import the images through the bulk import some images seem to not have the correct position. Since the function `fix_exif_orientation` (https://github.com/sozialhelden/wheelmap/blob/master/config/initializers/carrierwave.rb#L4-L10) exists we expected all uploaded images (manually and through bulk import) to display the images as the original EXIF orientation. But this is unfortunately not the case.

Needs further investigation why some images do not display correctly even if the original exif orientation shows a correct position.

Example node: https://wheelmap.org/nodes/4116869491

Screenshot Example:

![00000375](https://user-images.githubusercontent.com/4077916/28162692-47009500-67c8-11e7-88db-8e2bee7b9411.png)

**Comment by 1000miles** [comment]
### Update, 2017-07-2017

Looking at the download links of streetspotr our hunch is now that the problem with the incorrect image positions happen `before` or `on` upload on the streetspotr server side, e.g. , e.g. https://download.streetspotr.com/d2c6686b0931b938026e94254667a4fd1eef7118/c8a0e7b79ced3df0328ba306232bc05e7a5c6c99.

If our assumption is correct, we can not do anything here because we fetch these images as they are uploaded by streetspotr. @Svenyo Could you please talk to streetspotr and let streetspotr confirm our hunch? Merci. 

We will put this ticket `on hold` meanwhile.

![00000376](https://user-images.githubusercontent.com/4077916/28163958-752e161e-67cd-11e7-828f-dc8a9d43b115.jpg)


cc @holgerd @anngreenberg

**Comment by Svenyo** [comment]
@1000miles I forwarded this to Streetspotr. Will let you know when there's an answer.

**Comment by 1000miles** [comment]
@Svenyo Before we start another streetspotr import with #651 , do you know if Streetspotr has fixed this issue  (images upside down) meanwhile? 

cc @anngreenberg

**Comment by Svenyo** [comment]
@1000miles They confirmed that they had forwarded this bug to the IT team and fixed it, yes. Let's see if it works! :-)

**Comment by 1000miles** [comment]
@Svenyo cc @holgerd @anngreenberg 

We checked the newest streetspotr Sept 2017 batch from ticket #651 . Unfortunately the image distortion is not fixed yet. 

You can view examples here:

`DE_2016.csv`:

Download URLs:
- https://download.streetspotr.com/5e921e868dac49be0cf076e7e5eacf4614f854e8/d761020cb1008ee33bd066fda42fe57d7bdf303a
- https://download.streetspotr.com/67a6f4c3f3f797d4dc1954b2970a48f97d8f1cb0/3aa1a7113514acb762a16276dc68c935164ad178

Production POI:
- https://wheelmap.org/nodes/2956247446

**Comment by 1000miles** [comment]
A few more examples where the exif orientation issue still exist.

**Photos uploaded via streetspotr on 2017-09-11:**

- https://wheelmap.org/nodes/-128973881
- https://wheelmap.org/nodes/668074037
- https://wheelmap.org/nodes/-128973881

**Comment by Svenyo** [comment]
@1000miles: Yes, the problem was solved on August 11 from the part of the Streetspotr IT, so a lot of pics taken before that date are still displayed in a rotated view. After August 11 all pics are displayed correctly. So, at least for the next campaign, this problem will be solved.

**Comment by anngreenberg** [comment]
As @Svenyo mentioned, for future contributions the problem will be solved. 
Therefore closing.
